build.gradle

```diff
@@ -56,10 +60,10 @@ dependencies {
   compile 'org.ow2.asm:asm:5.0.4'
   compile 'org.ow2.asm:asm-commons:5.0.4'
   compile 'org.ow2.asm:asm-tree:5.0.4'
-  compile 'org.elasticsearch:elasticsearch:7.4.2'
+  compile 'org.elasticsearch:elasticsearch:7.5.0'
```

Comment:
I think we need to bump org.apache.lucene:lucene-expressions to 8.3.0

Reply:
Thanks will do